src/Carbon/Carbon.php

```diff
@@ -88,7 +88,7 @@ class Carbon extends DateTime implements JsonSerializable
     const MONTHS_PER_YEAR = 12;
     const MONTHS_PER_QUARTER = 3;
     const WEEKS_PER_YEAR = 52;
-    const DAYS_PER_MONTH = 30;
+    const WEEKS_PER_MONTH = 4;
```

Comment:
Even this is not ok :(

Reply:
If we want do be perfectly strict, 24 hours/day is not exact, 52 weeks/years is also an approximation.

It's OK for me to have approximated default values if they can be set by the user easily.

It's pretty obvious than with an interval of 3 months, getting totalDays or 3 weeks getting totalMonths, we cannot provide an exact value.